force-app/main/default/classes/ChatterPost_Utility.cls

```diff
@@ -0,0 +1,72 @@
+/**
+ * @description       : An helper class for Chatter Post Assignment
+ * @author            : Deepak Prajapati (d.prajapati@concret.io)
+ * @last modified on  : 05-10-2021
+ * @last modified by  : Deepak Prajapati (d.prajapati@concret.io)
+**************************/
+public with sharing class ChatterPost_Utility {
+
+    /******
+     * Wrapper for store users and emails 
+     **********************/
+    public class FollowersWrapper{
+         public List<String> emails = new List<String>();
+         public Set<Id> users = new Set<Id>();
+    }
+
+    /**********************************************
+    * @description Get Followers Details in a Grouped form
+    * @author Deepak Prajapati (d.prajapati@concret.io) | 04-10-2021 
+    * @param Set<Id> parentIds ----> In This Case Account Ids
+    * @return Map<Id, FollowersWrapper> ----> Id: AccountId, FollowerWrapper: That Contains userid list and follower emails 
+    *************************************************/
+    public static Map<Id, FollowersWrapper> getFollowersByAccount(Set<Id> parentIds){
+
+        Map<Id, FollowersWrapper> followersMap = new Map<Id, FollowersWrapper>();
+        Map<Id, User> userMap = new Map<Id, User>([SELECT Id, Email FROM User LIMIT 999]);
```

Comment:
what if your subscribers doesn't come under these 999 records?

Reply:
wouldn't it be better if a set of subscriber ids is created and then those users will automatically filter the query.